cypress/integration/todo_spec.js

```diff
@@ -55,13 +60,24 @@ describe('TodoMVC - React', () => {
   it('edits an existing todo', () => {
     addTodoItemInLocalStorage()
     todoItems()
-      .dblclick()
-      .find('.edit')
-      .clear()
-      .type(EDITED_ITEM)
-      .type('{enter}')
+    .dblclick()
+    .find('.edit')
+    .clear()
+    .type(EDITED_ITEM)
+    .type('{enter}')
 
     todoItems()
-      .should('contain', EDITED_ITEM)
+    .should('contain', EDITED_ITEM)
+  })
+
+  it('delete the todo', () => {
+    addTodoItemInLocalStorage()
+    todoItems()
+    .trigger('mouseover').get('.destroy')
+    .invoke('show').should('be.visible')
+    .click('right')
```

Comment:
Great work. Single question - is there any reason to click button at right corner?